### PR TITLE
libdispatch: new package

### PR DESCRIPTION
**Pull request by qmfrederik**
PR for upstream changes is here: https://github.com/apple/swift-corelibs-libdispatch/pull/814

### CODE REVIEW

**Comment by ognevny** [comment]
thanks! this package actually conflicts with https://packages.msys2.org/base/mingw-w64-libblocksruntime

**Comment by qmfrederik** [comment]
Maybe we should just remove the old libblocksruntime package (which is based off a tarball in the Debian package repositories, which I _think_ contains an old copy of the BlocksRuntime as part of LLVM's compiler-rt project) and use the libblocksruntime which ships with Swift instead?  That version has the advantage of being actively maintained.

**Comment by ognevny** [comment]
> Maybe we should just remove the old libblocksruntime package (which is based off a tarball in the Debian package repositories, which I _think_ contains an old copy of the BlocksRuntime as part of LLVM's compiler-rt project) and use the libblocksruntime which ships with Swift instead?  That version has the advantage of being actively maintained.

maybe. what do you think, @lazka?

File: mingw-w64-libdispatch/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  declare -a extra_config
+  if check_option "debug" "n"; then
+    extra_config+=("-DCMAKE_BUILD_TYPE=Release")
+  else
+    extra_config+=("-DCMAKE_BUILD_TYPE=Debug")
+  fi
+
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+    LDFLAGS="-fuse-ld=lld" \
```
Comment (ognevny) [inline mingw-w64-libdispatch/PKGBUILD:40]:
```suggestion
    -LDFLAGS="-fuse-ld=lld" \
    +LDFLAGS=" -fuse-ls=lld" \
```

**Comment by lazka** [comment]
> maybe. what do you think, @lazka?

sure, why not

**Comment by lazka** [comment]
thanks!

**Pull request merged**